js/app.js

```diff
@@ -52,12 +80,17 @@
    * to set the context to the correct object
    */
 
+  btn2.addEventListener('click', () => {
+    setMyLightGreen.apply(btn2);
+  })
 
   /*
    * Add a click event listener to btn3
    * the handler method will be a reference to the setMyLightGreen function
    */
-
+  btn3.addEventListener('click', () => {
```

Comment:
This is not actually doing what the instructions says. You don't need an anonymous function here.